AppOne/AppOne/View/HomeVC.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+final class HomeVC: UIViewController {
+    private lazy var brandsCollection: UICollectionView = {
+        let collectionV = UICollectionView(
+            frame: view.bounds,
+            collectionViewLayout: UICollectionViewCompositionalLayout.list(using: UICollectionLayoutListConfiguration(appearance: .plain)))
+        return collectionV
```

Comment:
you miss the `translatesAutoresizingMaskIntoConstraints`

Reply:
Ok moved it to here